docs/templates/layout.html

```diff
@@ -108,3 +113,20 @@
     }
   </style>
 {% endblock %}
+
+{% block footer %}
+{{ super() }}
+<div class="footer">This page uses <a href="https://analytics.google.com/">
+Google Analytics</a> to collect statistics. You can disable it by blocking
+the JavaScript coming from www.google-analytics.com.
```

Comment:
Can you add a link to the privacy policy here? I think it will be useful for user. It makes life easier.

Reply:
Done